manifests/deis-logger-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
I believe this needs to be updated with some more fields from the current version existing in the workflow-dev chart for `make upgrade` to work:
```
 kind: Deployment
 metadata:
   name: deis-logger
+  namespace: deis
   labels:
     heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: canary
 spec:
   replicas: 1
   strategy:
```